Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -2049,16 +2161,35 @@ def get_system_status(client, command_args) -> Tuple[str, dict, Union[list, dict
     ipaddrs_server = argToList(arg=command_args.get('ip_server'))
     port = arg_to_number(arg=command_args.get('port'))
 
-    is_resp_filtering_required = statuses or hostnames or ipaddrs_client or ipaddrs_client or ipaddrs_server or port
+    # is_resp_filtering_required = statuses or hostnames or ipaddrs_client or ipaddrs_client or ipaddrs_server or port
+    is_resp_filtering_required = statuses or ipaddrs_client or ipaddrs_client or ipaddrs_server or port
```

Comment:
How can we make sure it won't break expected response for customers? We might need to schedule a call regarding this. It is a very popular integration and we do not want to break anything.

Reply:
We can definitely schedule a call if needed! I tested the change on a few different hosts and each time was successful. If it works better, I can record another demo of just this change!